src/popup/config/config.json

```diff
@@ -338,6 +338,44 @@
               "sideloadUrl": "https://localhost:17159/intuneworkflows"
             }
           ]
+        },
+        {
+          "name": "Microsoft_Azure_CloudPC",
+          "environments": [
+            {
+              "label": "Int",
```

Comment:
Not a blocking comment: While it's fine to use any name that you'd like, naming it `Dogfood` will keep it grouped with all of the other `Dogfood` extensions.

Reply:
As CloudPC is a new extension and INT is our internal dev testing environment, integrated with Intune externsion's SH environment. It's not dogfood extension, so I prefer to INT.